### ISSUE TITLE
300 epoch time

### ISSUE THREAD

**Issue by alicera** [ultralytics/yolov5#4572]
## ❔Question
When using --resume, the log will be the 93 epochs completed .
How to let it show 300 epoch, even it resume 
93 epochs completed in 29.744 hours.

**Comment by glenn-jocher** [comment]
@alicera 👋 Hello! Thanks for asking about **resuming training**. YOLOv5 🚀 Learning Rate (LR) [schedulers](https://github.com/ultralytics/yolov5/blob/9f3a388cea66ca71b9bbe057929eed8fbc920a7e/train.py#L109-L142) follow predefined LR curves for the fixed number of `--epochs` defined at training start (`default=300`), and are designed to fall to a minimum LR on the final epoch for best training results. For this reason you can not modify the number of epochs once training has started.

<img width="850" alt="LR Curves" src="https://user-images.githubusercontent.com/26833433/119332195-8e113a80-bc88-11eb-8eb7-7494ff1d5c01.png">

If your training was **interrupted** for any reason you may continue where you left off using the `--resume` argument. If your training fully completed, you can start a **new** training from any model using the `--weights` argument. Examples:

## Resume Single-GPU

You may not change settings when resuming, and **no additional arguments** other than `--resume` should be passed, with an optional path to the checkpoint you'd like to resume from. If no checkpoint is passed the most recently updated `last.pt` in your `yolov5/` directory is automatically found and used:
```bash
python train.py --resume  # automatically find latest checkpoint (searches yolov5/ directory)
python train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Resume Multi-GPU

[Multi-GPU DDP trainings](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training) must be resumed with the same GPUs and DDP command, i.e. assuming 8 GPUs:
```bash
python -m torch.distributed.run --nproc_per_node 8 train.py --resume  # resume latest checkpoint
python -m torch.distributed.run --nproc_per_node 8 train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Start from Pretrained

If you would like to **start training** from a fully trained model, use the `--weights` argument, not the `--resume` argument:
```bash
python train.py --weights path/to/best.pt  # start from pretrained model
```

Good luck and let us know if you have any other questions!

**Comment by alicera** [comment]
Sorry, my question is

I have the data 10000. I train it with 300 epoch and it cost 10 hours.
If I train the yolo and stop it at 150 epoch. Then I resume it form 150 epoch. When training is done, it will log the 150 epochs completed in 5 hours.

How to know the cost time for 300 epoch ?

**Comment by glenn-jocher** [comment]
@alicera I don't understand your question. If 150 epochs takes x, then 2 * 150 will take 2 * x.

**Comment by alicera** [comment]
Assume that I use ctrl + c to stop the training until 150 epoch. So I don't know the training time  when 150 epoch. 
 I stop the training at 150 epoch. Then resume it. Finally, the log print the "150 epochs completed in 5 hours".

How can I get the training time for 300 epoch?

**Comment by glenn-jocher** [comment]
@alicera I think I'm not understanding. If it takes 5 hours to do something, and you want to do twice that, then you know what the time will be. 

We are extremely busy and do not have resources to address trivial questions, please do not ask questions you can solve yourself.
